### DISCUSSION TITLE
如何显示主屏幕

### DISCUSSION THREAD

**Discussion by Moli-X** [sumatrapdfreader/sumatrapdf#3152] [Q&A]
如何显示主屏幕

**Comment by GitHubRulesOK** [comment]
> **How to display the home screen ?**

(from google !)
显示主页，是一个高级设置 

![image](https://user-images.githubusercontent.com/12754913/205032111-13ab1046-992f-4533-820f-72eb00905eab.png)
...
ShowToc = true
**NoHomeTab = false**
TocDy = 0
...

![image](https://user-images.githubusercontent.com/12754913/205031173-27e741e9-da6b-469e-98b0-202945a05f3c.png)
![image](https://user-images.githubusercontent.com/12754913/205031660-3e432087-bab6-43f3-89af-52c9a587c56e.png)
